user_docs/en/userGuide.t2t

```diff
@@ -359,10 +363,7 @@ Once you navigate to an object, you can review its content using the [text revie
 When [Focus Highlight #VisionFocusHighlight] is enabled, the location of the current navigator object is also exposed visually.
 By default, the navigator object moves along with the System focus, though this behavior can be toggled on and off.
 
-Note that braille follows both the [focus #SystemFocus] and [caret #SystemCaret] as well as object navigation and text review by default.
-If you want it to follow the focus and caret only, you need to [configure braille to be tethered to #BrailleTether] focus.
-In this case, braille will not follow object navigation and text review.
-If you want braille to follow object navigation and text review instead, you need to [configure braille to be tethered to #BrailleTether] review.
+Note that by default, braille tries to follow automatically both the system focus and the NVDA navigator. This can be configured in the [Braille Tether section #BrailleTether].

```

Comment:
Sorry to be picky again @Adriani90, but this word order: "follow automatically" is a bit strange in English. I'd remove this whole line and specify the default only once in the #BrailleTether section.
Note also, two sentences should be two lines.

```suggestion
Note: Braille following Object Navigation can be configured via [Braille Tether #BrailleTether].
```